tests/framework/e2e/cluster.go

```diff
@@ -639,18 +638,23 @@ func (epc *EtcdProcessCluster) CloseProc(ctx context.Context, finder func(EtcdPr
 		return fmt.Errorf("failed to find member ID: %w", err)
 	}
 
+	memberRemoved := false
 	for i := 0; i < 10; i++ {
-		_, err = memberCtl.MemberRemove(ctx, memberID)
-		if err != nil && strings.Contains(err.Error(), rpctypes.ErrGRPCUnhealthy.Error()) {
-			time.Sleep(500 * time.Millisecond)
-			continue
+		_, err := memberCtl.MemberRemove(ctx, memberID)
+		if err != nil && strings.Contains(err.Error(), "member not found") {
+			memberRemoved = true
+			break
 		}
-		break
+
+		time.Sleep(500 * time.Millisecond)
 	}
-	if err != nil {
-		return fmt.Errorf("failed to remove member: %w", err)
+
+	if !memberRemoved {
+		return fmt.Errorf("failed to remove member after 10 tries")
```

Comment:
Minor comment: if there is no parameter, then you'd better use `errors.New` instead.

Reply:
fixed